projects/plugins/jetpack/modules/likes/queuehandler.js

```diff
@@ -170,14 +165,6 @@ function JetpackLikesMessageListener( event ) {
 			break;
 		}
 
-		case 'showCommentLikeWidget': {
-			const placeholder = document.querySelector( `#${ data.id } .likes-widget-placeholder` );
-			if ( placeholder ) {
-				placeholder.style.display = 'none';
-			}
-			break;
-		}
```

Comment:
I'd recomment to keep the `showCommentLikeWidget` case although it will be empty. I believe it will be soon populated again.

Here's why: currently the `jetpack-likes-widget-loaded` class is set a bit too soon, after the like iframe loads (`load` event listener). But at this moment the iframe doesn't have any content yet. The `likes-iframe` and its `likes-rest-nojquery` script will add DOM markup there, after loading the likes from REST, and only then it will send the `show(Comment)LikeWidget` message back to `queuehandler`. And it's the handler for this message that should set the `jetpack-likes-widget-loaded`.

We should do this in one of followup PRs and then `showCommentLikeWidget` will be handled again.

Reply:
I'm not very fond of keeping the empty block, but I get your point and added a comment why it's there, linking to this thread.